simtools/job_submission/job_manager.py

```diff
@@ -104,7 +104,7 @@ def submit(
 
         """
         self.run_script = str(run_script)
-        self.run_out_file = str(run_out_file)
+        self.run_out_file = str(run_out_file).replace(".log", "-log")
```

Comment:
Wait, but isn't `log` here the actual extension and therefore should be `.log`? Sorry if I annoy with this point, just want to make sure.

Reply:
OK now that I read below I actually see that there's another extension which is `.out`. We can chat about it tomorrow quickly, but in that case we probably don't need `-log` at all.

Reply:
OK - removed the log for out/err.